src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
 description: >
   Run all tests in a MATLAB project and generate test artifacts. MATLAB includes any files in your
   project that have a Test label. If your pipeline does not leverage a MATLAB project, then MATLAB
-  includes all tests in the the root of your repository including its subfolders.
+  includes all tests in the the root of your repository including its subfolders. The command uses the
+  topmost MATLAB version on the system path.
```

Comment:
Ok, I see we have the path info here too. So it's only run-build that is missing it.